CONTRIBUTING.md

```diff
@@ -38,57 +38,46 @@ Finally, set the upstream remote to the official Spyder repo with:
 $ git remote add upstream https://github.com/spyder-ide/spyder.git
 ```
 
+
 ### Creating a conda environment or virtualenv
 
-If you use Anaconda you can create a conda environment with the following commands:
+If you use Anaconda you can create a conda environment and install the necessary dependencies as follows.
 
 ```bash
-$ conda create -n spyder-dev python=3
+$ conda create -n spyder-dev -c conda-forge --file requirements/conda.txt
 $ conda activate spyder-dev
+(spyder-dev) $ python install_dev_repos.py --editable
 ```
 
-You can also use `virtualenv` on Linux, but `conda` is **strongly** recommended:
-
-```bash
-$ mkvirtualenv spyder-dev
-$ workon spyder-dev
-```
-
-
-### Installing dependencies
-
-After you have created your development environment, you need to install Spyder's necessary dependencies. The easiest way to do so (with Anaconda) is
+If you are running on macOS, you will also need to install `python.app`, so the first line above should read
 
 ```bash
-$ conda install -c conda-forge --file requirements/conda.txt
+$ conda create -n spyder-dev -c conda-forge --file requirements/conda.txt python.app
 ```
 
-This installs all Spyder's dependencies into the environment.
-If you are running on macOS, you will also need to install `python.app`.
+You can also use `virtualenv` on Linux, but `conda` is **strongly** recommended:
 
 ```bash
-$ conda install python.app
+$ mkvirtualenv spyder-dev
+$ workon spyder-dev
+(spyder-dev) $ install -e .
```

Comment:
```suggestion
(spyder-dev) $ pip install -e .
```

This seems like a typo?

Reply:
should we also advise `python -m pip`?

Reply:
I normally do when outside an activated venv, but the venv, so `pip` already points to the `pip` in the venv, otherwise something is very wrong.